src/main/resources/schema.sql

```diff
@@ -155,3 +155,16 @@ CREATE TABLE qotw_submissions (
 	  reviewed BOOL NOT NULL DEFAULT FALSE,
 	  accepted BOOL NOT NULL DEFAULT FALSE
 );
+
+CREATE TABLE help_account (
+	  user_id BIGINT PRIMARY KEY,
+	  experience DOUBLE NOT NULL,
+);
+
+CREATE TABLE help_transaction (
+	  id BIGINT PRIMARY KEY AUTO_INCREMENT,
+	  recipient BIGINT NOT NULL,
+	  created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+	  value DOUBLE NOT NULL,
+	  message VARCHAR(255)
```

Comment:
I am quite unsure if we really want to store up to 255 characters per transaction as there would be a pretty high amount of those.